bindings/py/src/nupic/bindings/regions/PyRegion.py

```diff
@@ -1,406 +0,0 @@
-# ----------------------------------------------------------------------
-# Numenta Platform for Intelligent Computing (NuPIC)
-# Copyright (C) 2013-2015, Numenta, Inc.  Unless you have an agreement
-# with Numenta, Inc., for a separate license for this software code, the
-# following terms and conditions apply:
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Affero Public License version 3 as
-# published by the Free Software Foundation.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
-# See the GNU Affero Public License for more details.
-#
-# You should have received a copy of the GNU Affero Public License
-# along with this program.  If not, see http://www.gnu.org/licenses.
-#
-# http://numenta.org/licenses/
-# ----------------------------------------------------------------------
-
-"""
-This file defines the base class for Python regions.
-"""
-
-import numpy
-import collections
-
-RealNumpyDType = numpy.float32
-from abc import ABCMeta, abstractmethod
-
-class DictReadOnlyWrapper(collections.Mapping):
-  """
-  Provides read-only access to a dict. When dict items are mutable, they can
-  still be mutated in-place, but dict items can't be reassigned.
-  """
-
-  def __init__(self, d):
-    self._d = d
-
-  def __iter__(self):
-    return iter(self._d)
-
-  def __len__(self):
-    return len(self._d)
-
-  def __getitem__(self, key):
-    return self._d[key]
-
-class PyRegion(object):
```

Comment:
this file is still important for NAPI, right?

Reply:
yes, you are correct. I'll add the whole folder.